css/styles.css

```diff
@@ -256,6 +408,14 @@ i {
   .heart-icon-details {
     font-size: 1.4rem;
   }
+
+  .error-message-data {
+    font-size: 1.2rem;
+  }
+
+  .lds-spinner {
+    left: 42%;
+  }
```

Comment:
Instead of doing varying `left` values, you should put this into a flexbox container and use `justify-content` to center this, so you don't need to keep adjusting this value.